instrumentation-core/spring/autoconfigure/src/main/java/io/opentelemetry/spring/autoconfigure/trace/TracerAutoConfiguration.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.opentelemetry.spring.autoconfigure.trace;
+
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.exporters.logging.LoggingSpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.SpanProcessor;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+import io.opentelemetry.trace.Tracer;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableConfigurationProperties(TracerProperties.class)
+@ConfigurationProperties(prefix = "opentelemetry.trace.tracer")
+public class TracerAutoConfiguration {
+
+  @Bean
+  @Autowired
+  public Tracer tracer(TracerProperties tracerProperties) throws Exception {
+    Tracer tracer = OpenTelemetry.getTracer(tracerProperties.getName());
+    setLoggingExporter(tracerProperties);
+    return tracer;
+  }
+
+  private void setLoggingExporter(TracerProperties tracerProperties) {
+    if (tracerProperties.isLoggingExpoerterIsEnabled()) {
```

Comment:
I'm don't think I follow. Here I setup a default logging exporter that prints the contents of a span to console after it's closed. The idea is to give users visibility into how the tracing is configured. This will be useful in the initial setup/debugging process but user's will eventually add their own span exporter to visualize traces. User's can disable this feature by adding `opentelemetry.tracer.loggingExpoerterIsEnabled=False` to their src/main/resourses/application.properties file. I'm not sure if this is best way to go about this